udemy_enroller/scrapers/comidoc.py

```diff
@@ -44,11 +44,19 @@ async def get_links(self) -> List:
 
         :return: List of udemy course urls
         """
+        
         comidoc_links = []
         self.current_page += 1
         coupons_data = await get(f"{self.DOMAIN}/coupons?page={self.current_page}")
+        
+        
         soup = BeautifulSoup(coupons_data.decode("utf-8"), "html.parser")
-        for course_card in soup.find_all("div", class_="MuiPaper-root"):
+        
+        with open("output1.html", "w", encoding='utf-8') as file:
+            file.write(str(soup))
```

Comment:
Not sure why we're writing a file here

Reply:
it's trash... :( I just save the page to map it on scrap.

Reply:
Removed :)